test_utils/utils.go

```diff
@@ -70,7 +70,7 @@ func WipeK8sPods() {
 	}
 }
 
-func RunIPAMPlugin(netconf, command, args string) (types.Result, int) {
+func RunIPAMPlugin(netconf, command, args string) (types.Result, types.Error, int) {
```

Comment:
@tomdee actually, it's used here: https://github.com/projectcalico/cni-plugin/pull/260/files#diff-470a64869ec059ff9ca8b16cd6f8ac06R138